ops/storage-ddls/cassandra-setup.cql

```diff
@@ -140,6 +140,27 @@ CREATE INDEX ON computedtopics (period);
 CREATE INDEX ON computedtopics (computedfeatures);
 CREATE INDEX ON computedtopics (periodtype);
 
+CREATE TABLE topplaces (
+    placeid text,
+    placename text,
+    periodstartdate timestamp,
+    periodenddate timestamp,
+    periodtype text,
+    period text,
+    mentions int,
+    PRIMARY KEY ((placeid), periodstartdate, periodenddate)
```

Comment:
The columns in the parens are the "table key" and the other ones are the "partition key". [More information](https://stackoverflow.com/a/24953331/3817588).